### PR TITLE
feat(core): add order summary page

### PR DESCRIPTION
**Pull request by bc-alexsaiannyi**
## What/Why?
This PR adds Order Summary (details) page for functionality. It should be mentioned that:

- it doesn't contain mutations & User actions (return or repurchase) related to them,
- no payment info for Order,
- layout supports both single & multi- consignments,
- no images for Order items.

## Testing
 For demo purpose multi-consignment order has been mocked

https://github.com/user-attachments/assets/0607fdbf-8dfd-4119-a8fe-11ee83f0e185

https://github.com/user-attachments/assets/fdb98468-e3f1-4df9-b365-f73f3adebbf5

### CODE REVIEW

File: core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+const OrderState = async ({ orderState }: { orderState: OrderDetailsDataType['orderState'] }) => {
+  const locale = await getLocale();
+  const t = await getTranslations({ locale, namespace: 'Account.Orders' });
+  const format = await getFormatter();
+  const { orderId, orderDate, status } = orderState;
+
+  return (
+    <div className="mb-6 flex flex-col gap-6 md:flex-row">
+      <div>
```
Comment (yurytut1993) [inline core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx:20]: can we remove this `div`?

Comment (bc-alexsaiannyi) [reply core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx:20]: I wrap order number & date in separate flex item via div.

File: core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  const t = await getTranslations({ locale, namespace: 'Account.Orders' });
+  const { orderState, summaryInfo, consignments } = data;
+  const isMultiShippingConsignments = consignments.shipping && consignments.shipping.length > 1;
+
+  const orderContent = consignments.shipping
+    ? consignments.shipping.map(({ lineItems }) => ({
+        lineItems: removeEdgesAndNodes(lineItems),
+      }))
+    : null;
+
```
Comment (yurytut1993) [inline core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx:250]: 🍺 
```
const orderContent = consignments.shipping?.map(({ lineItems }) => ({
  lineItems: removeEdgesAndNodes(lineItems),
}));
```

File: core/app/[locale]/(default)/account/(tabs)/orders/_components/product-snippet.tsx [hunk 3]
```diff
@@ -142,10 +144,15 @@ export const ProductSnippet = ({
...
-          ) : (
-            <div className="h-full w-full bg-gray-200" />
-          )}
-        </div>
+          </div>
+        )}
+        {!isExtended && defaultImage?.url !== 'string' && (
+          <div className="relative aspect-square flex-auto">
+            <div className="flex h-full w-full items-center justify-center bg-gray-200 text-gray-500">
+              <span>Coming Soon</span>
```
Comment (yurytut1993) [inline core/app/[locale]/(default)/account/(tabs)/orders/_components/product-snippet.tsx:152]: should be added via lang file?

**Review by yurytut1993 — APPROVED** [review]
👍

File: core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx (hunk 2 above)
Comment (bc-alexsaiannyi) [reply core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx]: Hah, thanks!🙈

**Pull request merged**